nodeJS/authentication/Security-Configuration.md

```diff
@@ -24,4 +24,5 @@ Digging into this package can be useful when creating bigger projects where app
 1. The main important takeaway here is to KEEP YOUR SECRETS SECRET by never accidentally publishing them.
 2. Read through the documentation for [dotenv](https://github.com/motdotla/dotenv#readme) and [nconf](https://github.com/indexzero/nconf). There are other ways to go about hiding your secrets, but these two projects are popular and widely used.
 3. Go back to your earlier projects and SECURE THEM!
+4. (Optional) If you still want to know more about environment variables [this](https://www.twilio.com/blog/working-with-environment-variables-in-node-js-html) article digs deeper into variations you might encounter moving forward.
```

Comment:
I don't think we want to add this to the Assignment section. 


However, this lesson is missing an additional resources section. If you would like to look at other lessons that have that, and add an "Additional Resources" section here, and add this link to it, you can. 

Sorry for the delayed response!